static/card-creator-video-tutorial.srt

```diff
@@ -0,0 +1,200 @@
+1
+00:00:00,000 --> 00:00:03,055
+Hi! I'm John. I'm a developer on Wordbots.
```

Comment:
Wow you made subtitles?

Reply:
Youtube made a rough draft, but I had to edit them to make them reasonable.